test/ProgressOnderwijsUtilsTests/TransactedLocalConnection.cs

```diff
@@ -1,23 +1,28 @@
 ﻿using System;
 using System.Data.SqlClient;
-using System.Transactions;
 using ProgressOnderwijsUtils;
 
 namespace ProgressOnderwijsUtilsTests
 {
     public class TransactedLocalConnection : IDisposable
     {
         public readonly SqlCommandCreationContext Context;
-        public readonly CommittableTransaction Transaction;
+#if NET461
+        public readonly System.Transactions.CommittableTransaction Transaction = new System.Transactions.CommittableTransaction();
+#endif
 
         public TransactedLocalConnection()
         {
-            Transaction = new CommittableTransaction();
             Context = new SqlCommandCreationContext(new SqlConnection(Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? @"Server = (localdb)\MSSQLLocalDB; Integrated Security = true"), 60, SqlCommandTracer.CreateAlwaysOffTracer());
             try {
                 Context.Connection.Open();
                 ParameterizedSql.TableValuedTypeDefinitionScripts.ExecuteNonQuery(Context);
+#if NET461
                 Context.Connection.EnlistTransaction(Transaction);
+#else
+                // .NET Core does not support EnlistTransaction
+                SafeSql.SQL($"begin transaction").ExecuteNonQuery(Context);

```

Comment:
is dit een idee om altijd zo te doen, en dus nooit met CommittableTransaction?  Of maakt dat de bulk copy stuk op .net framework?

Reply:
Dat maakt inderdaad bulkcopy stuk, die ondersteund alleen SqlTransaction. Het lijkt ook min of meer toevallig dat CommitableTransaction wel werkt, want dan accepteert hij ook nog de optie InternalTransaction en dat doet hij niet als je een SqlTransaction specificeert.